scripts/migration/migrate_date_modified_for_existing_nodes.py

```diff
@@ -0,0 +1,37 @@
+"""
+This will add a date_modified field to all nodes.  Date_modified will be equivalent to the date of the last log.
+"""
+import sys
+import logging
+
+from website import models
+from website.app import init_app
+
+logger = logging.getLogger(__name__)
+
+def date_updated(self):
+        """
```

Comment:
This block appears to be over-indented (should be 4 spaces), but Python doesn't seem to mind..?